oaknational/Oak-Web-Application
# Feat/lesq 269/navbar restyle [LESQ-269]

## Description

- Added a new `StyledHeader` component separate from `FixedHeader` currently in use
- Use the new component in `AppHeader` and `SiteHeader` -  are there any other places it should be updated?
- Update the background and brush underline for header and make not fixed to the top


## How to test

1. Go to https://deploy-preview-1931--oak-web-application.netlify.thenational.academy
2. You should see the newly styled app bar on the home page and on /teachers pages

## Screenshots

How it used to look (delete if n/a):
<img width="500" alt="Screenshot 2023-09-26 at 10 47 00" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/ae588a6e-be0a-4e2b-b371-0a5f254f60fd">
<img width="500" alt="Screenshot 2023-09-26 at 10 46 47" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/bd3b0e72-4f09-4488-aa73-08d87315784a">

How it should now look:
<img width="500" alt="Screenshot 2023-09-26 at 10 46 27" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/1ec663f9-4a20-4bea-819e-0fa3a13c804f">
<img width="500" alt="Screenshot 2023-09-26 at 10 46 14" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/df34d130-85f1-4678-a949-bf7f0ecd5be0">


## Checklist
- [ ]  White nav bar everywhere in OWA, black brush border
- [ ]  Not sticky anymore
- [ ]  Not the teachers/pupils link behaviour, that is in LESQ-270
